colin/cli/colin.py

```diff
@@ -48,6 +49,7 @@ def cli():
 @click.command(name="check",
                context_settings=CONTEXT_SETTINGS)
 @click.argument('TARGET', type=click.STRING)
+@click.argument('parent_target', type=click.STRING, required=False)
```

Comment:
IIRC we agreed to pass output of skopeo inspect  (which is in json)
Do we expect that `parent_target` will contain path to json file or string with json?

I do not have any preferences. I can pass anything to colin :-).
I just cannot see conversion string(json) -> object in PR.
But I check RP very briefly.

Reply:
If I understand that correctly, this is a different approach -- we have a name of the parent and handle it in the same way, as we are handling the "main" target.

I am not against this. It can probably have some overhead with handling the parent image, but it is more general so we can create other checks with that as well.

I also have no problem with getting info from JSON.

Reply:
> If I understand that correctly, this is a different approach -- we have a name of the parent and handle it in the same way, as we are handling the "main" target.

I am fine with that but it really need to be handled the same way as `target` in `class OstreeTarget`.
And I cannot see any validation for `parent target there`. lines 328 ..335

In another words, it has to work also with ostree format

e.g.
```
colin --target-type=ostree --parent-target ostree:${parentImage}@/var/tmp/os3/ ostree:${imageToTest}@/var/tmp/os3/
```

Reply:
it works for both ImageTarget and OstreeTarget

both of them are now accepting parent_target